### ISSUE TITLE
Update New Features and Update Documents in V0.2.6

### ISSUE THREAD

**Issue by Lupin1998** [Westlake-AI/openmixup#25]
Updating features:
1. Support new self-supervised method [BEiT](https://arxiv.org/abs/2106.08254) with ViT-Base on ImageNet-1K, and fix bugs of [CAE](https://arxiv.org/abs/2202.03026), [MaskFeat](https://arxiv.org/abs/2112.09133), and [SimMIM](https://arxiv.org/abs/2111.09886) in `Dataset`, `Model`, and `Head`. Note that we added `HOG` feature implementation borrowed from the [original repo](https://github.com/facebookresearch/SlowFast) for [MaskFeat](https://arxiv.org/abs/2112.09133).
2. Support new backbone architecture [DeiT-3](https://arxiv.org/abs/2204.07118) and provide configs.
3. Update pre-training and fine-tuning config files, and documents for the relevant masked image modeling (MIM) methods ([BEiT](https://arxiv.org/abs/2106.08254), [MaskFeat](https://arxiv.org/abs/2111.06377), [CAE](https://arxiv.org/abs/2202.03026), and [A2MIM](https://arxiv.org/abs/2205.13943)).
4. Support Grad-CAM visualization tools [vis_cam.py](tools/visualizations/vis_cam.py) of supported architectures.

Updating documents:
1. Update the template and add the latest paper lists of mixup and MIM methods in [Awesome Mixups](docs/en/awesome_selfsup/MIM.md) and [Awesome MIM](docs/en/awesome_selfsup/MIM.md).
2. Update [documents](docs/en/tools) of `tools`.

**Comment by Lupin1998** [comment]
Fixed bugs and updated features for MIM methods in OpenMixup V0.2.6.

**Comment by Lupin1998** [comment]
Update new features and documents for OpenMixup V0.2.6 (issue #24).
